build/root/WORKSPACE

```diff
@@ -1,15 +1,15 @@
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "441e560e947d8011f064bd7348d86940d6b6131ae7d7c4425a538e8d9f884274",
-    strip_prefix = "rules_go-c72631a220406c4fae276861ee286aaec82c5af2",
-    urls = ["https://github.com/bazelbuild/rules_go/archive/c72631a220406c4fae276861ee286aaec82c5af2.tar.gz"],
+    sha256 = "7e748c5fa17248eff598081bcb711172350335e6c4baefafb828fac3a6740f4f",
+    strip_prefix = "rules_go-6d900bc95ae678bec5c91031b8e987957d2a7f93",
+    urls = ["https://github.com/bazelbuild/rules_go/archive/6d900bc95ae678bec5c91031b8e987957d2a7f93.tar.gz"],
```

Comment:
what made you choose this version? I wonder if we could instead bump to f4bebf54dca2ad198b0311fd772c8100149cd647 (for go 1.9.2) while some of the performance issues are being worked out.